### PR TITLE
Imaqal graphs

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Add Imaqal Incoming, Outgoing and Failed sms graphs. Pending https://github.com/AfricasVoices/Operations-Dashboard/pull/34 
image

### CODE REVIEW

File: public/media_engagement/imaqal_media_engagement.js [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+// Append total sent sms graph to svg
+var total_failed_sms_graph = d3.select(".total_failed_sms_graph").append("svg")
+    .attr("width", Width + Margin.left + Margin.right)
+    .attr("height", Height + Margin.top + Margin.bottom)
+    .append("g")
+    .attr("transform",
+        "translate(" + Margin.left + "," + Margin.top + ")");
+
+// Format TimeStamp  
+var timeFormat = d3.timeFormat("%d %H:%M:%S");
```
Comment (lukechurch) [inline public/media_engagement/imaqal_media_engagement.js]: Please format as day/month/year

You can do this in a following PR if you wish

File: public/media_engagement/imaqal_media_engagement.js [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+        .attr("class", "line")
+        .style("stroke", "#935529")
+        .attr("d", total_somtel_received_line);
+    total_telesom_received_path.data([data])
+        .attr("class", "line")
+        .style("stroke", "#2A4B7C")
+        .attr("d", total_telesom_received_line);
+    
+    // update path data for total outgoing sms(s)
+    total_sent_path.data([data])
```
Comment (lukechurch) [inline public/media_engagement/imaqal_media_engagement.js:259]: As discussed lets factor into a dictionary

**Review by lukechurch — APPROVED** [review]
Let's follow up with some refactoring

File: public/media_engagement/imaqal_media_engagement.js (hunk 1 above)
Comment (IsaackMwenda) [reply public/media_engagement/imaqal_media_engagement.js]: Updated to hour/day/month/year

File: public/media_engagement/imaqal_media_engagement.js (hunk 2 above)
Comment (IsaackMwenda) [reply public/media_engagement/imaqal_media_engagement.js:259]: Thanks will refactor in a future pr.

**Pull request merged**